### DISCUSSION TITLE
Add custom/my link to navbar or sidebar menu

### DISCUSSION THREAD

**Discussion by Omnimes** [avitorio/outstatic#236] [Ideas]
While working with the Outstatic CMS, I miss the option to return to my main panel or to the homepage.
I would like to suggest adding such an option to Outstatic.

Maybe in a dropdown menu :)
![image](https://github.com/avitorio/outstatic/assets/166378457/f7aefc1d-2af8-4a1a-8390-a6b8c9ec6e4f)

**Comment by avitorio** [comment]
Hey,
Thanks for the suggestion.

What do you mean by homepage? Your website url? What would be the main panel?

**Comment by Omnimes** [comment]
Hi,

Let the homepage be our domain (homepage URL).

As for the panel, I meant that we should be able to add any link that we could add ourselves.

Currently, returning from outstatic to my admin panel is done by entering the URL. It would be more convenient to have such a link in outstatic.

**Reply by avitorio** [reply]
Hey,
This is a good suggestion. Thank you. I'll consider adding it to v2.
